addons/web_editor/static/src/js/editor/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -29,45 +29,60 @@ import {
     createDOMPathGenerator,
 } from '../utils/utils.js';
 
-Text.prototype.oDeleteBackward = function (offset, alreadyMoved = false) {
-    const parentNode = this.parentNode;
-
-    if (!offset) {
-        // Backspace at the beginning of a text node is not a specific case to
-        // handle, let the element implementation handle it.
-        HTMLElement.prototype.oDeleteBackward.call(this, offset, alreadyMoved);
-        return;
-    }
-
-    // Get the size of the unicode character to remove.
-    const charSize = [...this.nodeValue.slice(0, offset)].pop().length;
-    // Split around the character where the backspace occurs.
-    const firstSplitOffset = splitTextNode(this, offset - charSize);
-    const secondSplitOffset = splitTextNode(parentNode.childNodes[firstSplitOffset], charSize);
-    const middleNode = parentNode.childNodes[firstSplitOffset];
+/**
+ * Handle text node deletion for Text.oDeleteForward and Text.oDeleteBackward.
+ *
+ * @param {int} charSize
+ * @param {int} offset
+ * @param {DIRECTIONS} direction
+ * @param {boolean} alreadyMoved
+ */
+export function textDeletion(charSize, offset, direction, alreadyMoved) {
```

Comment:
`textDeletion` -> `deleteText`

Also, I'd put it in deleteForward actually, since technically "delete" is "deleteForward" as "deleteBackward" is backspace, but that's nitpicking.